### ISSUE TITLE
Integration tests for AI Workflows

### ISSUE THREAD

**Issue by dsilvam** [dotCMS/core#27960]
### Parent Issue

https://github.com/dotCMS/core/issues/27835

### Task

- Write automated tests for AI Workflow Sub-Actions
    - [x] Test for AI Auto-Tag Content Sub-Action
    - [x] Test for AI Content Prompt Sub-Action
    - [x] Test for AI Embeddings Sub-Action
    - [x] Test for AI Generate Image Sub-Action

### Proposed Objective

Core Features

### Proposed Priority

Priority 2 - Important

### Acceptance Criteria

- [ ] Automated test coverage for all Sub-Actions

**Comment by dsilvam** [comment]
Blocked by a upcoming refactor. <Refactor Epic issue # >

**Comment by john-thomas-dotcms** [comment]
We're going to delay this work until after we refactor the dotAI workflow actions to remove the save from them.

**Comment by jdotcms** [comment]
The AI wf are already covered by previous tasks, mark as a done!
